### PR TITLE
v0.12.0 - Refactor

### PR DESCRIPTION
**Pull request by NastuzziSamy**

### CODE REVIEW

**Comment by NastuzziSamy** [comment]
@rullzer @MorrisJobke @skjnldsv it is ok for you ? :)

File: appinfo/info.xml [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-      xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
+<info xmlns:xsi= "http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
     <id>files_rightclick</id>
-    <name>Files Right Click</name>
-    <summary>Add a rightclick menu generator for any Nextcloud apps</summary>
-    <description><![CDATA[This app will allow users but also developers to have a right click menu. Simply use the RightClick object to create quickly and easly menus. Already files app got a right click menu]]></description>
+    <name>Right click</name>
+    <summary>Right click menu for Nextcloud</summary>
+    <description><![CDATA[This app allows users and developers to have a right click menu. Simply use the RightClick object to quickly create context menus. The Files app already shows the actions menu when right clicking on files and folders.]]></description>
     <version>0.11.1</version>
```
Comment (MorrisJobke) [inline appinfo/info.xml]: Version not bumped.

**Comment by MorrisJobke** [comment]
Should we add `<default_enable>` here as well? Only problem: for the appstore release that entry in the `info.xml` needs to be removed. Otherwise publishing on the appstore doesn't work.

File: appinfo/info.xml (hunk 1 above)
Comment (NastuzziSamy) [reply appinfo/info.xml]: Fixed :)

**Comment by NastuzziSamy** [comment]
I can update my Makefile to fix that issue

**Comment by NastuzziSamy** [comment]
@MorrisJobke it should be good now

File: appinfo/info.xml [hunk 2]
```diff
@@ -18,4 +18,5 @@
     <dependencies>
         <nextcloud min-version="10" max-version="17" />
     </dependencies>
+    <default_enabled />
```
Comment (MorrisJobke) [inline appinfo/info.xml]: It's `default_enable`.

Comment (NastuzziSamy) [reply appinfo/info.xml]: ^^'

**Comment by MorrisJobke** [comment]
Starting tomorrow it will be in the master dailies and then in 16.

**Pull request merged**